javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/index.html

```diff
@@ -2,124 +2,80 @@
-                  />
-                </g>
-              </svg>
-            </button>
+          <h1 class="fcc-title">freeCodeCamp</h1>
+          <div>
+            <img src="assets/decor-lines.svg" alt="decoration lines right" />
           </div>
-
-          <ul class="playlist-container"></ul>
+          <div>
+            <img src="assets/menu.svg" alt="menu button" />
+          </div>
+        </div>
+        <div class="player__content">
+          <div id="player__album-art">
+            <img
+              src="https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg"
+              alt="song cover art"
+            />
+          </div>
+          <div class="player__display">
+            <div class="player__display--songArtist">
+              <p id="player__song-title">Cruising for a Musing</p>
+              <p id="player__song-artist">Quincy Larson</p>
+            </div>
+            <div class="player__buttons">
+              <button class="rewind">
+                <img src="assets/rewind.svg" alt="rewind button" />
+              </button>
+              <button class="play">
+                <img src="assets/play.svg" alt="play button" />
+              </button>
+              <button class="forward">
+                <img src="assets/forward.svg" alt="forward button" />
+              </button>
+              <button class="shuffle">
+                <img src="assets/shuffle.svg" alt="shuffle button" />
+              </button>
+              <button class="repeat">
+                <img src="assets/loop.svg" alt="repeat button" />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="playlist">
+        <div class="playlist__bar">
+          <div>
+            <img src="assets/decor-lines.svg" alt="decoration lines left" />
+          </div>
+          <h1 class="playlist-title">Playlist</h1>
```

Comment:
```suggestion
          <h2 class="playlist-title">Playlist</h2>
```

Generally there should only be one `h1` on a page. I think an `h2` makes sense here since the playlist is a subset of the player.